### PR TITLE
Falco plugin logger

### PR DESCRIPTION
**Pull request by SysdigDan**
Adding new logger for Falco OSS. Falco plugin is currently available here - https://github.com/SysdigDan/dnscollector-falco-plugin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ Additionally, DNS-collector also support
     - [`ElasticSearch`](doc/loggers.md#elasticsearch-client)
     - [`Scalyr`](doc/loggers.md#scalyr-client)
     - [`Redis`](doc/loggers.md#redispub)
+    - [`Falco`](doc/loggers.md#falco)
```
Comment (dmachard) [inline README.md:53]: Can you add this Falco logger in a sub item (Security tools ?)

File: loggers/falco.go [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+	listChannel := []chan dnsutils.DnsMessage{}
+	listChannel = append(listChannel, f.channel)
+	subprocessors := transformers.NewTransforms(&f.config.OutgoingTransformers, f.logger, f.name, listChannel)
+
+	for dm := range f.channel {
+		// apply tranforms
+		if subprocessors.ProcessMessage(&dm) == transformers.RETURN_DROP {
+			continue
+		}
+
```
Comment (dmachard) [inline loggers/falco.go:81]: Do you plan to send dns messages in batch mode to be more efficient under high load ?

File: doc/loggers.md [hunk 3]
```diff
@@ -563,4 +564,18 @@ redispub:
   text-format: ""
   buffer-size: 100
   redis-channel: dns-collector
+```
+
+### Falco
+
+Falco plugin Logger
```
Comment (dmachard) [inline doc/loggers.md:571]: Perhaps it should be nice to have the link to the other plugin on falcon side ?

**Review by dmachard — CHANGES_REQUESTED** [review]
Thank to improve the capability of this collector.

File: doc/loggers.md (hunk 3 above)
Comment (SysdigDan) [reply doc/loggers.md:571]: Updated.

File: README.md (hunk 1 above)
Comment (SysdigDan) [reply README.md:53]: Updated.

File: loggers/falco.go (hunk 2 above)
Comment (SysdigDan) [reply loggers/falco.go:81]: In a future release.

**Review by dmachard — APPROVED** [review]
Look good to me

**Pull request merged**